src/test/java/games/strategy/engine/framework/systemcheck/LocalSystemCheckerTest.java

```diff
@@ -5,15 +5,14 @@
 
 import org.junit.Test;
 
-import com.google.common.base.Throwables;
 import com.google.common.collect.ImmutableSet;
 
 public class LocalSystemCheckerTest {
 
   private static final SystemCheck PASSING_CHECK = new SystemCheck("no op", () -> {
   });
   private static final SystemCheck FAILING_CHECK =
-      new SystemCheck("throws exception", () -> Throwables.propagate(new Exception("test")));
+      new SystemCheck("throws exception", () -> SystemCheckTest.throwWrappedInRuntimeException(new Exception("test")));
```

Comment:
One concrete test case calling a utility method in another concrete test case is kind of a smell.  There doesn't seem to be much going on here and in `SystemCheckTest`.  In fact, there doesn't seem to be a requirement in either of these tests that a checked exception needs to be wrapped in an unchecked exception for the tests to pass.  I think that just muddies the focus of the failure case tests; all they require is that an unchecked exception be thrown from the `Runnable`.

The following changes seem to produce identical behavior (and with two fewer lines :smile:):

```diff
diff --git a/src/test/java/games/strategy/engine/framework/systemcheck/LocalSystemCheckerTest.java b/src/test/java/games/strategy/engine/framework/systemcheck/LocalSystemCheckerTest.java
index ea400ce..9ea989a 100644
--- a/src/test/java/games/strategy/engine/framework/systemcheck/LocalSystemCheckerTest.java
+++ b/src/test/java/games/strategy/engine/framework/systemcheck/LocalSystemCheckerTest.java
@@ -11,8 +11,9 @@ public class LocalSystemCheckerTest {
 
   private static final SystemCheck PASSING_CHECK = new SystemCheck("no op", () -> {
   });
-  private static final SystemCheck FAILING_CHECK =
-      new SystemCheck("throws exception", () -> SystemCheckTest.throwWrappedInRuntimeException(new Exception("test")));
+  private static final SystemCheck FAILING_CHECK = new SystemCheck("throws exception", () -> {
+    throw new RuntimeException();
+  });
 
   @Test
   public void testPassingCase() {
diff --git a/src/test/java/games/strategy/engine/framework/systemcheck/SystemCheckTest.java b/src/test/java/games/strategy/engine/framework/systemcheck/SystemCheckTest.java
index 0d6eff4..03ba342 100644
--- a/src/test/java/games/strategy/engine/framework/systemcheck/SystemCheckTest.java
+++ b/src/test/java/games/strategy/engine/framework/systemcheck/SystemCheckTest.java
@@ -6,9 +6,6 @@ import static org.junit.Assert.assertThat;
 import org.junit.Test;
 
 public class SystemCheckTest {
-
-  private final Exception testException = new Exception("Testing");
-
   @Test
   public void testPassingSystemCheck() {
     final SystemCheck check = new SystemCheck("msg", () -> {
@@ -21,7 +18,9 @@ public class SystemCheckTest {
 
   @Test
   public void testFailingSystemCheck() {
-    final SystemCheck check = new SystemCheck("msg", () -> throwWrappedInRuntimeException(testException));
+    final SystemCheck check = new SystemCheck("msg", () -> {
+      throw new RuntimeException();
+    });
 
     assertThat(check.wasSuccess(), is(false));
     assertThat(check.getResultMessage(), is("msg: false"));
@@ -29,11 +28,9 @@ public class SystemCheckTest {
 
   @Test
   public void remembersAndReturnsExceptions() {
-    final SystemCheck check = new SystemCheck("msg", () -> throwWrappedInRuntimeException(testException));
+    final SystemCheck check = new SystemCheck("msg", () -> {
+      throw new RuntimeException();
+    });
     assertThat(check.getException().isPresent(), is(true));
   }
-
-  static void throwWrappedInRuntimeException(final Throwable throwable) {
-    throw new RuntimeException(throwable);
-  }
 }
```